tests/php/ZzDiagTest.php

```
@@ -0,0 +1,86 @@
+
+/**
+ * TEMPORARY diagnostic test.
+ *
+ * @package PerformanceOptimise\Tests
+ */
+class ZzDiagTest extends \PHPUnit\Framework\TestCase {
+	use WPPO_Test_Bootstrap;
+
+	private function make_cache(): Cache {
+		return ( new \ReflectionClass( Cache::class ) )->newInstanceWithoutConstructor();
+	}
```

PHPMD (2.15.0)

[error] The method make_cache is not named in camelCase. (undefined)

(CamelCaseMethodName)